python/lsst/ctrl/bps/drivers.py

```diff
@@ -129,10 +131,17 @@ def _init_submission_driver(config_file, **kwargs):
     else:
         _LOG.debug("Skipping submission checks.")
 
-    # make submit directory to contain all outputs
-    submit_path = config["submitPath"]
-    os.makedirs(submit_path, exist_ok=True)
-    config[".bps_defined.submitPath"] = submit_path
+    # Make submit directory to contain all outputs.
+    submit_path = Path(config["submitPath"])
+    try:
+        submit_path.mkdir(parents=True, exist_ok=False)
+    except OSError as exc:
+        if exc.errno == errno.EEXIST:
+            reason = "Directory already exists"
+        else:
+            reason = f"{exc.strerror}"
```

Comment:
Isn't the f-string unnecessary here?

Reply:
No, it isn't, will fix. (Originally, there was some additional text I got rid off eventually, but somehow forgot to remove the f-string.)